### DISCUSSION TITLE
Do we have community support channel? Discord or Slack

### DISCUSSION THREAD

**Discussion by DevanshuBrahmbhatt** [NVIDIA-NeMo/Guardrails#528] [Q&A]
I just wanted to know the ways I have to interact with the community to clear a few doubts.

**Comment by drazvan** [comment]
Hi @DevanshuBrahmbhatt! Currently, NeMo Guardrails doesn't have a dedicated discord/slack. The best route is Github Issues/Discussions.

**Reply by tonkatsu7** [reply]
> Hi @DevanshuBrahmbhatt! Currently, NeMo Guardrails doesn't have a dedicated discord/slack. The best route is Github Issues/Discussions.

NVIDIA has a Discord, why don't we just create a channel in there? https://discord.gg/GuPCZBnB

**Comment by DevanshuBrahmbhatt** [comment]
Ok Thanks
